src/components/FeeChart/FeeChart.tsx

```diff
@@ -0,0 +1,244 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  Flex,
+  Heading,
+  Slider,
+  SliderFilledTrack,
+  SliderMark,
+  SliderThumb,
+  SliderTrack,
+  Stack,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import type { Data, Layout, PlotHoverEvent } from 'plotly.js'
+import { useState } from 'react'
+import Plot from 'react-plotly.js'
```

Comment:
Can we add a react.lazy if the package is hefty?

Reply:
looking at yarn.lock theres a few deps so might be hefty overall if not the package itself